handout/testing/components/async.md

```diff
@@ -1,8 +1,8 @@
 # Testing Asynchronous Actions
 
-Some components rely on asynchronous actions to work. This can be tricky to test, as we don't know exactly when to look at the DOM for results. Fortunately Angular provides a function `fakeAsync` which essential fakes asynchronous behaviour. FakeAsync will wrap our test in a zone, and then listen for any asynchronous operations, like setTimeouts, Promises, callbacks, etc but will not actually call those functions asynchronously. Instead, it will rely on us calling the `tick` function to call those functions immediately and simulate time elapsing.
```

Comment:
The previous page in the book also tests an async action.  We can now use `whenStable()` for the.  FakeAsyc is useful for precise control over the passage of time.